src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int ACTION_INDEX = 0;
+    static final int VALUE_INDEX = 1;
+    static final String ACTION_SUPPLY = "supply";
+    static final String ACTION_BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/213884362-c95a244d-d271-4f97-9d91-258ccf461598.png)
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writetoFile(report, toFileName);
    }
```